app/coffeescripts/views/wiki/WikiPageDeleteDialog.coffee

```diff
@@ -15,7 +15,7 @@ define [
   class WikiPageDeleteDialog extends DialogFormView
     setViewProperties: false
     wrapperTemplate: -> '<div class="outlet"></div>'
-    template: -> I18n.t 'delete_confirmation', 'Are you sure you wish to delete this wiki page?'
+    template: -> ((@model.get('context_id') == '1)' ? 'warning' : I18n.t 'delete_confirmation', 'Are you sure you wish to delete this wiki page?')
```

Comment:
Did you mean to have this change in this PR?  If it's the warning about deleting pages from the Content LIbrary deleting it from synced courses, I think we need to update the language a bit.